[Mon Sep 10 15:12:12 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Unable to load ZF2. Run `php composer.phar install` or define a ZF2_PATH environment variable.' in /home/bondvt04/www/repository5x/init_autoloader.php:48\nStack trace:\n#0 /home/bondvt04/www/repository5x/public_html/index.php(9): include()\n#1 {main}\n  thrown in /home/bondvt04/www/repository5x/init_autoloader.php on line 48
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/Album/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/Album/Module.php on line 25
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/Album/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/Album/Module.php on line 25
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. boolean given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('Album', false)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\EventManage in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/Album/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/Album/Module.php on line 25
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/Album/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/Album/Module.php on line 25
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. boolean given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('Album', false)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\EventManage in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/Album/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/Album/Module.php on line 25
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/Album/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/Album/Module.php on line 25
[Thu Sep 13 10:31:55 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. boolean given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('Album', false)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\EventManage in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 13 10:32:47 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/Album/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/Album/Module.php on line 24
[Thu Sep 13 10:32:47 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/Album/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/Album/Module.php on line 24
[Thu Sep 13 10:33:11 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/Album/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/Album/Module.php on line 24
[Thu Sep 13 10:33:11 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/Album/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/Album/Module.php on line 24
[Thu Sep 13 11:40:00 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "album/album/index"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:451\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(230): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(196): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 451
[Thu Sep 13 13:39:26 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Album\\AlbumTable' not found in /home/bondvt04/www/repository5x/module/Album/Module.php on line 32
[Thu Sep 13 13:51:13 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Album\\AlbumTable' not found in /home/bondvt04/www/repository5x/module/Album/Module.php on line 32
[Thu Sep 13 14:00:45 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Album\\AlbumTable' not found in /home/bondvt04/www/repository5x/module/Album/Module.php on line 32
[Mon Sep 17 21:26:05 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Unable to load ZF2. Run `php composer.phar install` or define a ZF2_PATH environment variable.' in /home/bondvt04/www/repository5x/init_autoloader.php:48\nStack trace:\n#0 /home/bondvt04/www/repository5x/public_html/index.php(10): include()\n#1 {main}\n  thrown in /home/bondvt04/www/repository5x/init_autoloader.php on line 48
[Mon Sep 17 21:26:05 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Unable to load ZF2. Run `php composer.phar install` or define a ZF2_PATH environment variable.' in /home/bondvt04/www/repository5x/init_autoloader.php:48\nStack trace:\n#0 /home/bondvt04/www/repository5x/public_html/index.php(10): include()\n#1 {main}\n  thrown in /home/bondvt04/www/repository5x/init_autoloader.php on line 48
[Mon Sep 17 21:26:05 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Unable to load ZF2. Run `php composer.phar install` or define a ZF2_PATH environment variable.' in /home/bondvt04/www/repository5x/init_autoloader.php:48\nStack trace:\n#0 /home/bondvt04/www/repository5x/public_html/index.php(10): include()\n#1 {main}\n  thrown in /home/bondvt04/www/repository5x/init_autoloader.php on line 48
[Wed Sep 19 13:27:03 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'AdminIndex\\Controller\\Zend\\View\\HelperBroker' not found in /home/bondvt04/www/repository5x/module/AdminIndex/src/AdminIndex/Controller/IndexController.php on line 12
[Wed Sep 19 13:27:23 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'AdminIndex\\Controller\\Zend\\View\\HelperBroker' not found in /home/bondvt04/www/repository5x/module/AdminIndex/src/AdminIndex/Controller/IndexController.php on line 13
[Wed Sep 19 13:27:33 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'AdminIndex\\Controller\\Zend\\View\\HelperBroker' not found in /home/bondvt04/www/repository5x/module/AdminIndex/src/AdminIndex/Controller/IndexController.php on line 13
[Wed Sep 19 15:01:59 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for baseUrl' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('baseUrl', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('baseUrl', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('baseUrl')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(20): Zend\\View\\Renderer\\PhpRenderer->__call('baseUrl', Array)\n#4 /home/bondvt0 in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:02:04 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for baseUrl' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('baseUrl', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('baseUrl', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('baseUrl')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(20): Zend\\View\\Renderer\\PhpRenderer->__call('baseUrl', Array)\n#4 /home/bondvt0 in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:02:54 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for baseUrl' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('baseUrl', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('baseUrl', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('baseUrl')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(19): Zend\\View\\Renderer\\PhpRenderer->__call('baseUrl', Array)\n#4 /home/bondvt0 in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:03:58 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for baseUrl' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('baseUrl', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('baseUrl', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('baseUrl')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(19): Zend\\View\\Renderer\\PhpRenderer->__call('baseUrl', Array)\n#4 /home/bondvt0 in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:04:46 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined variable: view in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 20
[Wed Sep 19 15:04:46 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to a member function getBroker() on a non-object in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 20
[Wed Sep 19 15:04:57 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for getBroker' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('getBroker', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('getBroker', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('getBroker')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(20): Zend\\View\\Renderer\\PhpRenderer->__call('getBroker', Array)\n#4 /ho in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:06:04 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for getBroker' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('getBroker', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('getBroker', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('getBroker')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(20): Zend\\View\\Renderer\\PhpRenderer->__call('getBroker', Array)\n#4 /ho in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:06:41 2012] [error] [client 127.0.0.1] PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 131596288 bytes) in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 21
[Wed Sep 19 15:25:04 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for getHelper' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:420\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(108): Zend\\ServiceManager\\ServiceManager->get('getHelper', true)\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(333): Zend\\ServiceManager\\AbstractPluginManager->get('getHelper', NULL)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(351): Zend\\View\\Renderer\\PhpRenderer->plugin('getHelper')\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(20): Zend\\View\\Renderer\\PhpRenderer->__call('getHelper', Array)\n#4 /ho in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 420
[Wed Sep 19 15:40:55 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "hello"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:451\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('hello')\n#1 [internal function]: Zend\\View\\Helper\\Partial->__invoke('hello', Array)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): call_user_func_array(Object(Zend\\View\\Helper\\Partial), Array)\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->__call('partial', Array)\n#4 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->partial('hello', A in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 451
[Wed Sep 19 15:41:15 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "hello.phtml"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:451\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('hello.phtml')\n#1 [internal function]: Zend\\View\\Helper\\Partial->__invoke('hello.phtml', Array)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): call_user_func_array(Object(Zend\\View\\Helper\\Partial), Array)\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->__call('partial', Array)\n#4 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer-> in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 451
[Wed Sep 19 15:41:33 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "index.phtml"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:451\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('index.phtml')\n#1 [internal function]: Zend\\View\\Helper\\Partial->__invoke('index.phtml', Array)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): call_user_func_array(Object(Zend\\View\\Helper\\Partial), Array)\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->__call('partial', Array)\n#4 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer-> in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 451
[Wed Sep 19 15:42:22 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "index/hello.phtml"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:451\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('index/hello.pht...')\n#1 [internal function]: Zend\\View\\Helper\\Partial->__invoke('index/hello.pht...', Array)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): call_user_func_array(Object(Zend\\View\\Helper\\Partial), Array)\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->__call('partial', Array)\n#4 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Re in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 451
[Wed Sep 19 15:43:05 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to a member function setAttribute() on a non-object in /home/bondvt04/www/repository5x/module/Album/view/album/album/add.phtml on line 8
[Wed Sep 19 15:44:30 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "application/partials/hello.phtml"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:451\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('application/par...')\n#1 [internal function]: Zend\\View\\Helper\\Partial->__invoke('application/par...', Array)\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): call_user_func_array(Object(Zend\\View\\Helper\\Partial), Array)\n#3 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->__call('partial', Array)\n#4 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11 in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 451
[Wed Sep 19 15:47:18 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Argument 1 passed to Reflection::export() must implement interface Reflector, instance of Zend\\View\\Renderer\\PhpRenderer given, called in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 458 and defined in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 2
[Wed Sep 19 15:48:44 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Reflection::getName() in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 2
[Wed Sep 19 15:49:58 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined variable: function in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 6
[Wed Sep 19 15:49:58 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to a member function getName() on a non-object in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 6
[Wed Sep 19 15:50:24 2012] [error] [client 127.0.0.1] PHP Warning:  ReflectionClass::getName() expects exactly 0 parameters, 1 given in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 6
[Wed Sep 19 15:50:32 2012] [error] [client 127.0.0.1] PHP Warning:  ReflectionClass::getName() expects exactly 0 parameters, 1 given in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 6
[Wed Sep 19 15:51:12 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'ReflectionException' with message 'Class  does not exist' in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml:5\nStack trace:\n#0 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml(5): ReflectionClass->__construct('')\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(458): include('/home/bondvt04/...')\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('admin-index/ind...')\n#3 [internal function]: Zend\\View\\Helper\\Partial->__invoke('admin-index/ind...', Array)\n#4 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): call_user_func_array(Object(Zend\\View\\Helper\\Partial), Array)\n#5 /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml(11): Zend\\View\\Renderer\\PhpRenderer->__ in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 5
[Wed Sep 19 15:54:22 2012] [error] [client 127.0.0.1] PHP Parse error:  syntax error, unexpected '{', expecting ')' in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 3
[Wed Sep 19 15:54:28 2012] [error] [client 127.0.0.1] PHP Parse error:  syntax error, unexpected '->' (T_OBJECT_OPERATOR) in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 3
[Wed Sep 19 16:29:45 2012] [error] [client 127.0.0.1] PHP Warning:  ReflectionObject::__construct() expects parameter 1 to be object, null given in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 2
[Wed Sep 19 16:31:30 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Object of class Zend\\View\\Helper\\Partial could not be converted to string in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 24
[Wed Sep 19 16:32:25 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined variable: y in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 4
[Wed Sep 19 16:33:08 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined variable: y in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 4
[Wed Sep 19 17:31:48 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Zend_View_Helper_Doctype' not found in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 27
[Wed Sep 19 17:32:43 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\View\\Helper\\Doctype::doctype() in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 28
[Wed Sep 19 17:32:54 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\View\\Helper\\Doctype::doctype() in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 29
[Wed Sep 19 17:33:00 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\View\\Helper\\Doctype::doctype() in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 29
[Wed Sep 19 17:33:19 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Doctype' not found in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/index.phtml on line 28
[Wed Sep 19 17:34:15 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Zend_View_Helper_Doctype' not found in /home/bondvt04/www/repository5x/module/Album/view/album/album/index.phtml on line 5
[Wed Sep 19 17:35:08 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'AdminIndex\\Controller\\Zend_View_Helper_Doctype' not found in /home/bondvt04/www/repository5x/module/AdminIndex/src/AdminIndex/Controller/IndexController.php on line 12
[Wed Sep 19 17:35:24 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'AdminIndex\\Controller\\Zend_View_Helper_Doctype' not found in /home/bondvt04/www/repository5x/module/AdminIndex/src/AdminIndex/Controller/IndexController.php on line 13
[Wed Sep 19 17:35:34 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'AdminIndex\\Controller\\Doctype' not found in /home/bondvt04/www/repository5x/module/AdminIndex/src/AdminIndex/Controller/IndexController.php on line 13
[Thu Sep 20 22:25:51 2012] [error] [client 127.0.0.1] PHP Warning:  ReflectionObject::__construct() expects parameter 1 to be object, null given in /home/bondvt04/www/repository5x/module/AdminIndex/view/admin-index/index/hello.phtml on line 2
[Thu Sep 20 22:25:51 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/favicon.ico
[Thu Sep 20 22:25:51 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/favicon.ico
[Thu Sep 20 22:28:21 2012] [error] [client 127.0.0.1] PHP Fatal error:  Cannot redeclare class AdminIndex\\Module in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 6
[Thu Sep 20 22:28:29 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layouts/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfra in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:30:04 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layouts/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfra in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:30:24 2012] [error] [client 127.0.0.1] PHP Fatal error:  Cannot redeclare class AdminIndex\\Module in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 6
[Thu Sep 20 22:34:58 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-index/index/index"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(230): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(196): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewMo in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:36:48 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. integer given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('AdminCommon', 1)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\EventMana in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 20 22:37:10 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/AdminCommon/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 23
[Thu Sep 20 22:37:10 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/AdminCommon/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 23
[Thu Sep 20 22:37:10 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. boolean given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('AdminCommon', false)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\Event in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 20 22:37:11 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/AdminCommon/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 23
[Thu Sep 20 22:37:11 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/AdminCommon/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 23
[Thu Sep 20 22:37:11 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. boolean given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('AdminCommon', false)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\Event in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 20 22:37:23 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Exception\\RuntimeException' with message 'Module (AdminCommon) could not be initialized.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php:138\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php(80): Zend\\ModuleManager\\ModuleManager->loadModule('AdminCommon')\n#1 [internal function]: Zend\\ModuleManager\\ModuleManager->onLoadModules(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\EventManager->triggerListeners('loadModules', Object(Zend\\ModuleManager\\ModuleEvent), NULL)\n#4 /home/bondvt04/www/repository5x/vendor/zendframework/zendf in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php on line 138
[Thu Sep 20 22:37:50 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. NULL given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('AdminCommon', NULL)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\EventMana in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 20 22:38:09 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/AdminCommon/config/module.config.php): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 25
[Thu Sep 20 22:38:09 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/AdminCommon/config/module.config.php' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/module/AdminCommon/Module.php on line 25
[Thu Sep 20 22:38:09 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Listener\\Exception\\InvalidArgumentException' with message 'Config being merged must be an array, implement the \\Traversable interface, or be an instance of Zend\\Config\\Config. boolean given.' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php:330\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(147): Zend\\ModuleManager\\Listener\\ConfigListener->addConfig('AdminCommon', false)\n#1 [internal function]: Zend\\ModuleManager\\Listener\\ConfigListener->onLoadModule(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(209): Zend\\EventManager\\Event in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php on line 330
[Thu Sep 20 22:38:17 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-index/index/index"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(230): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(196): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewMo in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:41:04 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-index/index/index"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(230): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(196): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewMo in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:41:10 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-index/index/index"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(230): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(196): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewMo in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:43:38 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layouts/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfra in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:43:46 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layout/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfram in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:44:08 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layout/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfram in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:44:25 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "layout/layoutK"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendframework/zendfr in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:44:46 2012] [error] [client 127.0.0.1] PHP Warning:  include(/home/bondvt04/www/repository5x/module/Application/config/../view/layout/layout.phtml): failed to open stream: No such file or directory in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 463
[Thu Sep 20 22:44:46 2012] [error] [client 127.0.0.1] PHP Warning:  include(): Failed opening '/home/bondvt04/www/repository5x/module/Application/config/../view/layout/layout.phtml' for inclusion (include_path='.:/usr/share/pear') in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 463
[Thu Sep 20 22:47:49 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layout/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfram in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:48:43 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "admin-common/layout/layout"; resolver could not resolve to a file' in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:456\nStack trace:\n#0 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/View.php(203): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#1 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/Mvc/View/Http/DefaultRenderingStrategy.php(128): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#2 [internal function]: Zend\\Mvc\\View\\Http\\DefaultRenderingStrategy->render(Object(Zend\\Mvc\\MvcEvent))\n#3 /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(469): call_user_func(Array, Object(Zend\\Mvc\\MvcEvent))\n#4 /home/bondvt04/www/repository5x/vendor/zendfram in /home/bondvt04/www/repository5x/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 456
[Thu Sep 20 22:49:44 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/images/logo.jpg, referer: http://repository5x.bondvt04-home.lan/admin
[Thu Sep 20 22:49:44 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/images/logo.jpg, referer: http://repository5x.bondvt04-home.lan/admin
[Thu Sep 20 22:50:02 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/images/logo.jpg, referer: http://repository5x.bondvt04-home.lan/admin
[Thu Sep 20 22:50:02 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/images/logo.jpg, referer: http://repository5x.bondvt04-home.lan/admin
[Mon Oct 15 22:12:48 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'Unable to load ZF2. Run `php composer.phar install` or define a ZF2_PATH environment variable.' in /home/bondvt04/www/repository5x/init_autoloader.php:48\nStack trace:\n#0 /home/bondvt04/www/repository5x/public_html/index.php(10): include()\n#1 {main}\n  thrown in /home/bondvt04/www/repository5x/init_autoloader.php on line 48
[Mon Oct 15 22:12:48 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/favicon.ico
[Mon Oct 15 22:12:48 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/repository5x/public_html/favicon.ico
[Fri Oct 19 22:55:59 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Zend\\Debug\\Zend_Debug' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 19
[Fri Oct 19 22:55:59 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Fri Oct 19 22:55:59 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Fri Oct 19 23:03:26 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Application\\Controller\\Tasks' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 19
[Fri Oct 19 23:04:16 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\Tasks' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 21
[Fri Oct 19 23:04:49 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Exception\\RuntimeException' with message 'Module (Tasks) could not be initialized.' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php:139\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php(80): Zend\\ModuleManager\\ModuleManager->loadModule('Tasks')\n#1 [internal function]: Zend\\ModuleManager\\ModuleManager->onLoadModules(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(208): Zend\\EventManager\\EventManager->triggerListeners('loadModules', Object(Zend\\ModuleManager\\ModuleEvent), NULL)\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/Module in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php on line 139
[Fri Oct 19 23:05:06 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Exception\\RuntimeException' with message 'Module (Tasks) could not be initialized.' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php:139\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php(80): Zend\\ModuleManager\\ModuleManager->loadModule('Tasks')\n#1 [internal function]: Zend\\ModuleManager\\ModuleManager->onLoadModules(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(208): Zend\\EventManager\\EventManager->triggerListeners('loadModules', Object(Zend\\ModuleManager\\ModuleEvent), NULL)\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/Module in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php on line 139
[Fri Oct 19 23:06:11 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ModuleManager\\Exception\\RuntimeException' with message 'Module (Tasks) could not be initialized.' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php:139\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php(80): Zend\\ModuleManager\\ModuleManager->loadModule('Tasks')\n#1 [internal function]: Zend\\ModuleManager\\ModuleManager->onLoadModules(Object(Zend\\ModuleManager\\ModuleEvent))\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(468): call_user_func(Array, Object(Zend\\ModuleManager\\ModuleEvent))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/EventManager/EventManager.php(208): Zend\\EventManager\\EventManager->triggerListeners('loadModules', Object(Zend\\ModuleManager\\ModuleEvent), NULL)\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/Module in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ModuleManager/ModuleManager.php on line 139
[Fri Oct 19 23:10:13 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined variable: tasksModel in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 22
[Fri Oct 19 23:10:37 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Application\\Controller\\Tasks' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 21
[Fri Oct 19 23:12:12 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Tasks\\Model\\TasksModel::select() in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 22
[Fri Oct 19 23:12:17 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Tasks\\Model\\TasksModel::fetchAll() in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 22
[Fri Oct 19 23:13:07 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\TasksModel' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 21
[Fri Oct 19 23:15:17 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined property: Application\\Controller\\IndexController::$tasksTable in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 29
[Fri Oct 19 23:15:17 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\Tasks' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 17
[Fri Oct 19 23:15:33 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\Tasks' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 17
[Fri Oct 19 23:15:56 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\Tasks' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 17
[Fri Oct 19 23:16:17 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Application\\Controller\\TaskModel' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 23
[Fri Oct 19 23:16:22 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Application\\Controller\\TaskModel' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 23
[Fri Oct 19 23:16:30 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Application\\Controller\\TaskModel' not found in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 23
[Fri Oct 19 23:16:58 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\Tasks' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 17
[Fri Oct 19 23:17:30 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\TasksModel' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 17
[Sat Oct 20 13:43:58 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Sat Oct 20 13:43:58 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Sat Oct 20 14:39:28 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Db\\Adapter\\Driver\\Pdo\\Statement::current() in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 31
[Sat Oct 20 14:39:52 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Object of class Zend\\Db\\Adapter\\Driver\\Pdo\\Statement could not be converted to string in /home/bondvt04/www/tasker/module/Application/src/Application/Controller/IndexController.php on line 31
[Sat Oct 20 14:46:22 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62
[Sat Oct 20 14:53:22 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62
[Sat Oct 20 14:55:00 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to a member function getArrayCopy() on a non-object in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 37
[Sat Oct 20 15:05:06 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined variable: adapter in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 35
[Sat Oct 20 15:05:06 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to a member function createStatement() on a non-object in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 35
[Sat Oct 20 15:09:12 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Argument 1 passed to Zend\\Db\\Adapter\\Driver\\Pdo\\Statement::setParameterContainer() must be an instance of Zend\\Db\\Adapter\\ParameterContainer, string given, called in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Adapter.php on line 181 and defined in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Statement.php on line 141
[Sat Oct 20 15:32:23 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Argument 1 passed to Zend\\Db\\Adapter\\Driver\\Pdo\\Statement::setParameterContainer() must be an instance of Zend\\Db\\Adapter\\ParameterContainer, string given, called in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Adapter.php on line 181 and defined in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Statement.php on line 141
[Sat Oct 20 16:00:33 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62
[Sat Oct 20 16:02:39 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined offset: 1 in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/ParameterContainer.php on line 118
[Sat Oct 20 16:02:53 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62
[Sat Oct 20 16:03:20 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Db\\Adapter\\ParameterContainer::offsetSetFromArray() in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 36
[Sat Oct 20 16:04:05 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62
[Sat Oct 20 16:04:32 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62
[Sun Oct 21 19:59:21 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Sun Oct 21 19:59:21 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Mon Oct 22 21:27:05 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Mon Oct 22 21:27:05 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Mon Oct 22 22:03:50 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\Zend\\Debug\\Debug' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:04:56 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Db\\ResultSet\\ResultSet::getHello() in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:05:47 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Db\\ResultSet\\ResultSet::getHello() in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:06:42 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method ArrayObject::getHello() in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:11:51 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Db\\Adapter\\Driver\\Pdo\\Result::__toString() in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:12:01 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Object of class Zend\\Db\\Adapter\\Driver\\Pdo\\Result could not be converted to string in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:12:39 2012] [error] [client 127.0.0.1] PHP Catchable fatal error:  Object of class PDOStatement could not be converted to string in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TasksTable.php on line 30
[Mon Oct 22 22:27:07 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined property: Tasks\\Model\\TaskModel::$task in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TaskModel.php on line 27
[Mon Oct 22 22:27:07 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined property: Tasks\\Model\\TaskModel::$task in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TaskModel.php on line 27
[Tue Oct 23 18:53:09 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Tue Oct 23 18:53:09 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Tue Oct 23 18:59:09 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\DefaultStrategy' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TaskModel.php on line 11
[Tue Oct 23 19:59:39 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Stdlib\\Hydrator\\ClassMethods::setObject() in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TaskModel.php on line 76
[Tue Oct 23 22:51:17 2012] [error] [client 127.0.0.1] PHP Notice:  Undefined property: Tasks\\Model\\MySuperClass::$ololo in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Model/TaskModel.php on line 63
[Sun Oct 28 21:33:54 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62, referer: http://tasker.bondvt04.home.lan/tasks/add
[Sun Oct 28 21:35:20 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62, referer: http://tasker.bondvt04.home.lan/tasks/add
[Sun Oct 28 21:38:56 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62, referer: http://tasker.bondvt04.home.lan/tasks/add
[Sun Oct 28 21:39:08 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62, referer: http://tasker.bondvt04.home.lan/tasks/add
[Sun Oct 28 21:39:32 2012] [error] [client 127.0.0.1] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Debug/Debug.php:122) in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Http/PhpEnvironment/Response.php on line 62, referer: http://tasker.bondvt04.home.lan/tasks/add
[Sun Oct 28 21:48:07 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Model\\TasksTable' not found in /home/bondvt04/www/tasker/module/Tasks/Module.php on line 34
[Sun Oct 28 21:48:07 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Sun Oct 28 21:48:07 2012] [error] [client 127.0.0.1] File does not exist: /home/bondvt04/www/tasker/public_html/favicon.ico
[Sun Oct 28 21:53:00 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\View\\Exception\\RuntimeException' with message 'Zend\\View\\Renderer\\PhpRenderer::render: Unable to render template "task-row.phtml"; resolver could not resolve to a file' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php:454\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Helper/Partial.php(73): Zend\\View\\Renderer\\PhpRenderer->render('task-row.phtml')\n#1 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Helper/PartialLoop.php(70): Zend\\View\\Helper\\Partial->__invoke('task-row.phtml', Object(Tasks\\Entity\\TaskEntity))\n#2 [internal function]: Zend\\View\\Helper\\PartialLoop->__invoke('task-row.phtml', Object(Zend\\Db\\ResultSet\\HydratingResultSet))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(355): call_user_func_array(Object(Zend\\View\\Helper\\PartialLoop), Array)\n#4 /home/bondvt04/www/tasker/module/Defau in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php on line 454
[Sun Oct 28 21:53:22 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for getId' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:424\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(110): Zend\\ServiceManager\\ServiceManager->get('getId', true)\n#1 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(335): Zend\\ServiceManager\\AbstractPluginManager->get('getId', NULL)\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): Zend\\View\\Renderer\\PhpRenderer->plugin('getId')\n#3 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/task-row.phtml(2): Zend\\View\\Renderer\\PhpRenderer->__call('getId', Array)\n#4 /home/bondvt04/www/tasker/module/DefaultIndex/v in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 424
[Sun Oct 28 21:58:12 2012] [error] [client 127.0.0.1] PHP Fatal error:  Cannot access protected property Tasks\\Entity\\TaskEntity::$id in /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml on line 19
[Sun Oct 28 22:05:32 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Helper/PartialLoop.php(66): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 [internal function]: Zend\\View\\Helper\\PartialLoop->__invoke('default-index/i...', Object(Zend\\Db\\ResultSet\\HydratingResultSet))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(355): call_user_func_array(Object(Zend\\View\\Helper\\PartialLoop), Array)\n#4 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(29): Ze in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:08:59 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to undefined method Zend\\Db\\ResultSet\\HydratingResultSet::getCount() in /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml on line 19
[Sun Oct 28 22:10:52 2012] [error] [client 127.0.0.1] PHP Fatal error:  Call to a member function current() on a non-object in /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml on line 19
[Sun Oct 28 22:12:52 2012] [error] [client 127.0.0.1] PHP Notice:  Trying to get property of non-object in /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml on line 28
[Sun Oct 28 22:12:52 2012] [error] [client 127.0.0.1] PHP Notice:  Trying to get property of non-object in /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml on line 28
[Sun Oct 28 22:14:02 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Entity\\Exception\\BadMethodCallException' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Entity/TaskEntity.php on line 54
[Sun Oct 28 22:15:22 2012] [error] [client 127.0.0.1] PHP Fatal error:  Class 'Tasks\\Entity\\Exception' not found in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Entity/TaskEntity.php on line 54
[Sun Oct 28 22:15:31 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Exception' with message 'The option "__strictMode__" does not have a matching getStrictMode getter method which must be defined' in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Entity/TaskEntity.php:54\nStack trace:\n#0 /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Entity/TaskEntity.php(32): Tasks\\Entity\\TaskEntity->__get('__strictMode__')\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(28): Tasks\\Entity\\TaskEntity->__set('task', 'Hello, world')\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#5 /home/bondvt04/www/tasker/vendor/zendfr in /home/bondvt04/www/tasker/module/Tasks/src/Tasks/Entity/TaskEntity.php on line 54
[Sun Oct 28 22:16:35 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:17:59 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:18:48 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:19:35 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:20:05 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:20:06 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:20:27 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:20:37 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:20:54 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:21:13 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:21:17 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:21:23 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:22:16 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:22:28 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:22:59 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:23:03 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:23:22 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(31): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:26:11 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(32): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:26:12 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(32): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:27:44 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\Adapter\\Exception\\RuntimeException' with message 'This result is a forward only result set, calling rewind() after moving forward is not supported' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php:159\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php(228): Zend\\Db\\Adapter\\Driver\\Pdo\\Result->rewind()\n#1 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(33): Zend\\Db\\ResultSet\\AbstractResultSet->rewind()\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\V in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/Adapter/Driver/Pdo/Result.php on line 159
[Sun Oct 28 22:29:28 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\ResultSet\\Exception\\RuntimeException' with message 'Buffering must be enabled before iteration is started' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php:100\nStack trace:\n#0 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(33): Zend\\Db\\ResultSet\\AbstractResultSet->buffer()\n#1 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(197): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\Vie in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php on line 100
[Sun Oct 28 22:29:32 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\Db\\ResultSet\\Exception\\RuntimeException' with message 'Buffering must be enabled before iteration is started' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php:100\nStack trace:\n#0 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/index.phtml(33): Zend\\Db\\ResultSet\\AbstractResultSet->buffer()\n#1 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(461): include('/home/bondvt04/...')\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(204): Zend\\View\\Renderer\\PhpRenderer->render(Object(Zend\\View\\Model\\ViewModel))\n#3 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(232): Zend\\View\\View->render(Object(Zend\\View\\Model\\ViewModel))\n#4 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/View.php(197): Zend\\View\\View->renderChildren(Object(Zend\\View\\Model\\Vie in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/Db/ResultSet/AbstractResultSet.php on line 100
[Sun Oct 28 22:30:52 2012] [error] [client 127.0.0.1] PHP Fatal error:  Uncaught exception 'Zend\\ServiceManager\\Exception\\ServiceNotFoundException' with message 'Zend\\ServiceManager\\ServiceManager::get was unable to fetch or create an instance for getId' in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php:424\nStack trace:\n#0 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ServiceManager/AbstractPluginManager.php(110): Zend\\ServiceManager\\ServiceManager->get('getId', true)\n#1 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(335): Zend\\ServiceManager\\AbstractPluginManager->get('getId', NULL)\n#2 /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/View/Renderer/PhpRenderer.php(353): Zend\\View\\Renderer\\PhpRenderer->plugin('getId')\n#3 /home/bondvt04/www/tasker/module/DefaultIndex/view/default-index/index/task-row.phtml(3): Zend\\View\\Renderer\\PhpRenderer->__call('getId', Array)\n#4 /home/bondvt04/www/tasker/module/DefaultIndex/v in /home/bondvt04/www/tasker/vendor/zendframework/zendframework/library/Zend/ServiceManager/ServiceManager.php on line 424
